[FIX WEBSITE-224] - Fix cron expression

michaelmccaskill opened the pull request:
Related to issue [WEBSITE-224]

Summary of this pull request: 

It seems I misread the cron expression format node-schedule supports. It
should now update every 15 minutes rather than 15 hours.

scherler commented:
@reviewbybees

reviewbybees commented:
This pull request originates from a [CloudBees](https://www.cloudbees.com/) employee. At CloudBees, we require that all pull requests be reviewed by other CloudBees employees before we seek to have the change accepted. If you want to learn more about our process please see [this explanation](https://github.com/reviewbybees/about#about-reviewbybees).

michaelneale commented on Dockerfile:
```diff
@@ -1,18 +1,17 @@
+COPY ./server/ /plugins/server/
 COPY ./app/ /plugins/app/
-COPY ./css/ /plugins/css/
+RUN cd /plugins/ && npm install
```
could put WORKDIR before RUN and then wouldn't need cd *nitpick*

scherler replied:
good catch I am pushing that now.

michaelneale commented:
:bee:

gusreiber commented:
:bee:

Commit:
[WEBSITE-224] - Fix cron expression

It seems I misread the cron expression format node-schedule supports. It
should now update every 15 minutes rather than 15 hours.

michaelmccaskill commented:
Per a discussion I had with Daniel, given the critical nature of this bug I am merging this myself.
